spec/filters/cidr_spec.rb

```diff
@@ -35,6 +36,75 @@
        insist { subject["tags"] }.nil?
     end
   end
+ 
+  # Test multple CIDR blocks passed into 'network'.  Make sure we try an
+  # IP in every range.
+
+  describe "IPV4 match, passing a list to network [192.168.0.30]" do
+   config <<-CONFIG
```

Comment:
we might be able to move this config out, as is the same for all describe sections.